Marlin/Conditionals_post.h

```diff
@@ -730,11 +730,16 @@
   /**
    * Set granular options based on the specific type of leveling
    */
+
+  #if ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(DELTA)
+    #define UBL_DELTA
```

Comment:
@oldmcg
Sorry this got merged before proper review.
You've made `UBL_DELTA` a conditional, but in parts of the code it's being treated like a feature. I'm doing cleanup on recent commits, and am inclined to return several parts of the code that have been changed to `ENABLED(UBL_DELTA)` back to `ENABLED(AUTO_BED_LEVELING_UBL)`. If the aim is to support UBL with delta, then simply do that, and use this a conditional that combines `ENABLED(AUTO_BED_LEVELING_UBL) && ENABLED(DELTA)` for conciseness. In that case, it makes no sense have sanity tests that "require" `UBL_DELTA` with DELTA and UBL. And in any case, some of the sanity tests are wrong. I'll give you a link to the patch when it's ready.